lua/entities/gmod_wire_expression2/core/damage.lua

```diff
@@ -132,6 +132,6 @@ E2Lib.registerEvent("entityDamage", {
 	{ "Damage", "xdm" }
 })
 
-hook.Add("EntityTakeDamage", "E2_entityDamage", function(victim, dmg)
+hook.Add("PostEntityTakeDamage", "E2_entityDamage", function(victim, dmg, took)
```

Comment:
You can probably get rid of the extra `, took` as it isnt used or change it to `, _took` to satisfy the linter